src/views/Viewer.vue

```diff
@@ -655,6 +657,20 @@ export default {
 			}
 		},
 
+		keyboardDownloadFile(event) {
+			if (event.key === 'd' && event.ctrlKey === true) {
```

Comment:
I've chosen ctrl+d here because ctrl+s would conflict with https://github.com/nextcloud/viewer/pull/1317

Reply:
Do we know if any other files manager uses shortcuts? If so, is there some kind of common agreement on the shortcut? :)

Reply:
I guess we could have a look at google drive & co what they use for this...

Reply:
google drive indeed uses ctrl+s for downloading a file from the viewer. However I disliike this conceptually as ctrl+s is more fitting for saving an edited document back to its storage just like the editor will do but this is only my personal opinion...

Reply:
Ctrl+d does not have any function in the google drive viewer...

Reply:
https://support.google.com/drive/answer/2563044?hl=en&co=GENIE.Platform%3DDesktop&oco=0

Reply:
Onedrive:

> Shift+F10 or Windows Menu key, then Down arrow key or Tab key until you reach the Download option, and then Enter.

Google drive does not have this shortcut :see_no_evil: 
I guess we can pick whatever we want then :grin:

Reply:
> https://support.google.com/drive/answer/2563044?hl=en&co=GENIE.Platform%3DDesktop&oco=0

this does not list the shortcuts for the viewer that is built into google drive... However I tested it and download in the viewer is ctrl+s but I find ctrl+d better...

Reply:
@jancborchardt would ctrl+d be fine for you? :)

Reply:
I mean, since ctrl+s only is active when the editor is opened, we could also use it for downloading :shrug: 
It will just not work when editing :wink: 

I personally have no preferences, I'll fall back behind whatever you choose (or Jan) :)

Reply:
Ctrl-S seems to be the most fitting here, it's also what I expected. And it's what Google does as you said. (If there is anyone using something already established, we should not invent our own. :)

We do indeed have to make sure it does not conflict with our Ctrl-S to save in Text and Office (and e.g. the new image editor), but for other filetypes like images, and videos, the shortcut is fine to download.

Reply:
fine by me!

Reply:
done